src/plugins/typedoc.ts

```diff
@@ -1,266 +0,0 @@
-                          type: this.resolveTypeString(param.type),
-                      };
-                  });
-        return {
-            documentation: this.renderComment(sig),
-            kind: Kind.Signature,
-            parameters,
-            returnType: this.resolveTypeString(sig.type),
-            type: this.resolveSignature(sig),
-        };
-    };
-
-    private resolveTypeString = (type: any): string => {
-        switch (type.type) {
-            case "array":
-                return this.resolveTypeString(type.elementType) + "[]";
-            case "reflection":
-                return this.resolveReflectionType(type.declaration);
-            case "union":
-                return type.types.map(this.resolveTypeString).join(" | ");
-            case "intersection":
-                return type.types.map(this.resolveTypeString).join(" & ");
-            default:
-                const name = type.name == null ? "?" : type.name;
-
-                if (name === "__type") {
-                    return "{}";
-                }
-
-                if (type.typeArguments) {
-                    const typeArgs = type.typeArguments.map(this.resolveTypeString).join(", ");
-                    return `${name}<${typeArgs}>`;
-                }
-                return name;
-        }
-    };
-
-    private resolveReflectionType = (decl: DeclarationReflection): string => {
-        if (decl.signatures) {
-            return decl.signatures.map(this.resolveSignature).join(" | ");
-        }
-        return "??";
-    };
-
-    private resolveSignature = (sig: SignatureReflection): string => {
-        const paramList = !sig.parameters
-            ? ""
-            : sig.parameters
-                  .map((param: any) => {
-                      const name = (param.flags && param.flags.isRest ? "..." : "") + param.name;
-                      const type = this.resolveTypeString(param.type);
-                      return `${name}: ${type}`;
-                  })
-                  .join(", ");
-        const returnType = this.resolveTypeString(sig.type);
-        return `(${paramList}) => ${returnType}`;
-    };
-
-    /**
-     * Converts a typedoc comment object to a rendered `IBlock`.
-     */
-    private renderComment = (obj: Reflection) => {
-        if (!obj || !obj.comment) {
-            return undefined;
-        }
-
-        const { comment } = obj;
-        let documentation = "";
-        if (comment.shortText) {
-            documentation += comment.shortText;
-        }
-        if (comment.text) {
-            documentation += "\n\n" + comment.text;
-        }
-        if (comment.tags) {
-            documentation += "\n\n" + comment.tags.map((tag: any) => `@${tag.tag} ${tag.text}`).join("\n");
-        }
-        return this.compiler.renderBlock(documentation);
-    };
-
-    /**
-     * Visit any object that has a matching `kind`. Recursively test
-     * children.
-     */
-    private visitKind<T>(
```

Comment:
this function has been replaced with some neat typedoc Reflection methods: `.getChildrenByKind()` and `.getReflectionsByKind()`